content/cpp/concepts/math-functions/terms/ceil/ceil.md

```diff
@@ -39,3 +39,21 @@ int main() {
   // Output: The result is 13!
 }
 ```
+
+## Codebyte Example
```

Comment:
Can we make the codebyte example a little more different than the prior example.